sdk/textanalytics/Azure.AI.TextAnalytics.Protocol/README.md

```diff
@@ -12,12 +12,307 @@ At this time, if you are looking to interact with the Azure Cognitive Services T
 
 ### Authenticate the client
 
+To create the client, you can use an `AzureKeyCredential`, with your API Key:
+
+```csharp
+TextAnalyticsClient client = new TextAnalyticsClient(new Uri("<endpoint-from-portal>"), new AzureKeyCredential("<api-key-from-portal>"));
+```
+
 ## Key concepts
```

Comment:
I think before we go into details of JsonData it would be good to state that client has methods that call the service and these methods take JSON payloads as inputs and return JSOn payloads as outputs. In typical SDK clients these payloads are represented by model types. In low-level clients these payloads are represented by a general purpose JsonData type.